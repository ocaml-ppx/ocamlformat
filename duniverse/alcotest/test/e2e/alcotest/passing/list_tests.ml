let () =
  let open Alcotest in
  let failtest () = invalid_arg "This test should never be run" in
  run ~argv:[| ""; "list" |] __FILE__
    [
      ( "test-a",
        [ test_case "alpha" `Quick failtest; test_case "beta" `Quick failtest ]
      );
      ( "test-b",
        [
          test_case "lorem ipsum dolor sit amet consectutor adipiscing elit"
            `Quick failtest;
        ] );
    ]
