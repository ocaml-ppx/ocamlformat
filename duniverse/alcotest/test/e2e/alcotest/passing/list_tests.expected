test-a          0   alpha.
test-a          1   beta.
test-b          0   lorem ipsum dolor sit amet consectutor adipiscing elit.
